cli/azd/pkg/azd/default.go

```diff
@@ -109,7 +108,6 @@ func (p *DefaultPlatform) ConfigureContainer(container *ioc.NestedContainer) err
 	container.MustRegisterSingleton(storage.NewBlobSdkClient)
 
 	// cosmosdb
-	ioc.RegisterInstance(container, &arm.ClientOptions{})
```

Comment:
@wbreza - Was there an intent to have injected `arm.ClientOptions` objects be empty? At this point I think it only affects `NewCosmosDbService`... but that basically puts the cosmos db service outside the configuration of the rest of the app.

Reply:
I don't know what this is.

Reply:
I'm going to assume it's OK to use the configuration in `container.go` instead of an empty `arm.ClientOptions` here. Tests have passed at this point.